packages/terra-legacy-theme/README.md

```diff
@@ -3,29 +3,112 @@
+    - [Format 1 Setup](#format-1-setup)
+    - [PostCSS Assets Plugin Webpack Config](#postcss-assets-plugin-webpack-config)
+  - [Format 2 - Scoped Theme](#format-2---scoped-theme)
+- [LICENSE](#license)
 
-### What deprecation means
-Any apps that currently use terra-legacy-theme will continue to work as-is.
-However, the terra-legacy-theme package will no longer continue to be supported / developed.
+## Getting Started
 
-## Why?
-When we started development, we had planned to use scss variables for colors, fonts, etc. Early in development, we switched to using CSS custom properties, though we left this package untouched. This package is not used in any terra-ui components, but we have yet to remove it from the repo. The variables provided in this package are not in sync with what is actually used in terra. The default theme provided by terra components is meant to be the legacy looking theme. We do not recommend using package.
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-legacy-theme`
 
-## How to replace this package?
-Instead of using this file, we've opted to hard-code these values as fallback values directly into the source code of our components using the CSS custom property syntax. This has eliminated the need for using this package in the terra-ui components. This approach may be applicable for teams as they move away from using this package as-well.
+## Usage
+This project offers two formats of terra-legacy-theme based around [browsers support for native CSS custom properties](http://caniuse.com/#search=custom%20properties) .
 
-~~The legacy-theme component sets global variables for the entire application. Global variables are included for responsive breakpoints, colors, and typography.~~
+* One that can be used in browsers that support native support for CSS custom properties as well as browsers that lack support for native CSS custom properties.
+* One that can only be used in browsers that support native support for CSS custom properties.
 
-- [Getting Started](#getting-started)
-- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-legacy-theme/docs)
-- [LICENSE](#license)
+It is recommended to use the first format when using terra-legacy-theme as your default theme.
 
-## Getting Started
+### Format 1 - Root Theme
+This theme format can be used to replace terra's default theme variables with custom theme variables and can be compiled turning the themed CSS custom properties into static values. The static values work in both browsers that lack support for native CSS custom properties and browsers that support CSS custom properties.
 
-- Install from [npmjs](https://www.npmjs.com): `npm install terra-legacy-theme`
+In your application, import the `terra-legacy-theme` module like so:
+
+```js
+// ES6 import
+import 'terra-legacy-theme';
+```
+
+#### Format 1 Setup
+
+You'll need to set up some PostCSS tools to ensure the terra-legacy-theme can be compiled correctly.
```

Comment:
How much of this is covered by Terra toolkit? It a lot of it is, it maybe better to specify using terra-toolkit?

Reply:
Added a note about using terra-toolkit: https://github.com/cerner/terra-core/pull/1467/commits/c7460970d56bb0b7176e3509cdc5dd37714c4ccf